### PR TITLE
NS-269 Generate section-wide GPA averages

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/NS-269

### CODE REVIEW

File: nessie/sql_templates/create_intermediate_schema.template.sql [hunk 1]
```diff
@@ -50,10 +50,6 @@ AS (
         en.enrollment_status != 'D'
 );
 
-/*
- * Copy S3 external data to Redshift table for faster client queries.
- */
```
Comment (paulkerschen) [inline nessie/sql_templates/create_intermediate_schema.template.sql]: This comment is duplicated above.

File: nessie/sql_templates/create_sis_schema.template.sql [hunk 2]
```diff
@@ -90,6 +90,20 @@ WITH SERDEPROPERTIES (
...
 LOCATION '{loch_s3_sis_data_path}/manifests/enrollments.json';
 
+-- term gpa
+CREATE EXTERNAL TABLE {redshift_schema_sis}.term_gpa
+(
+    sid VARCHAR,
+    term_id INT,
+    units_total DOUBLE PRECISION,
+    units_taken_for_gpa DOUBLE PRECISION,
+    gpa DOUBLE PRECISION,
```
Comment (johncrossman) [inline nessie/sql_templates/create_sis_schema.template.sql]: Minor: Trailing comma

Comment (paulkerschen) [reply nessie/sql_templates/create_sis_schema.template.sql]: Python habits!

**Comment by johncrossman** [comment]
Looks quite good. I'll merge when Pythonic comma is gone.

**Pull request merged**